vpr/src/route/route_timing.cpp

```diff
@@ -1444,6 +1454,10 @@ static float get_timing_driven_expected_cost(int inode, int target_node, float c
                                      + num_segs_ortho_dir * ortho_data.C_load)
                      + ipin_data.T_linear;
 
+        if (get_delay){
+		return Tdel;
+	}
+        
```

Comment:
Would it be possible to refactor this code to avoid the `get_delay` flag all together?

For instance adding a function `get_timing_driven_expected_delay()` which returns the expected delay to the target. `get_timing_driven_expected_delay()` could then be used by `get_timing_driven_expected_cost()`, and by other code which is currently setting the `get_delay` flag to `1`.